src/Sylius/Bundle/SettingsBundle/Resolver/ResolverServiceRegistry.php

```diff
@@ -30,7 +30,7 @@ class ResolverServiceRegistry extends ServiceRegistry
      */
     public function __construct($interface, SettingsResolverInterface $defaultResolver)
     {
-        parent::__construct($interface);
+        parent::__construct($interface, 'Settings resolver');
```

Comment:
It is possible to have more than one instance of this registry - so I would put it in the DI config. :+1: